<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>395</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>715</width>
    <height>395</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>715</width>
    <height>395</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestor de Proyectos</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>712</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QTableWidget" name="ActivTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>711</width>
      <height>321</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <column>
     <property name="text">
      <string>Actividad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Recursos</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Duraci√≥n</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Predecesores</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tiempo Tope</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Coste N.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Coste Op.</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="addButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/add.png</normaloff>icons/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="editButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/edit.png</normaloff>icons/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="deleteButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/minus.png</normaloff>icons/minus.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>268</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="reportButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../ProjectManagement-build-desktop-Qt_4_8_1_in_PATH__System__Debug/icons/report.png</normaloff>../ProjectManagement-build-desktop-Qt_4_8_1_in_PATH__System__Debug/icons/report.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="resourceButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/resources.png</normaloff>icons/resources.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="calendarButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/calendar.png</normaloff>icons/calendar.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="analysisButton">
       <property name="toolTip">
        <string>Analisis Flexibilidad</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../ProjectManagement-build-desktop-Qt_4_8_1_in_PATH__System__Debug/icons/analysis.png</normaloff>../ProjectManagement-build-desktop-Qt_4_8_1_in_PATH__System__Debug/icons/analysis.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo_Proyecto"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar_Proyecto"/>
    <addaction name="actionGuardar_Como"/>
    <addaction name="separator"/>
    <addaction name="actionCargar_Proyecto"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuInformes">
    <property name="title">
     <string>Opciones</string>
    </property>
    <widget class="QMenu" name="menuMostrar">
     <property name="title">
      <string>Mostrar</string>
     </property>
     <addaction name="actionModo_Simple"/>
     <addaction name="actionModo_Completo"/>
     <addaction name="actionPersonalizar"/>
    </widget>
    <addaction name="acGen_Informe"/>
    <addaction name="acGen_Ultimo"/>
    <addaction name="separator"/>
    <addaction name="menuMostrar"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionAcerca_de"/>
    <addaction name="actionAcerca_de_QT"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuInformes"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGuardar_Proyecto">
   <property name="text">
    <string>Guardar Proyecto</string>
   </property>
  </action>
  <action name="actionCargar_Proyecto">
   <property name="text">
    <string>Cargar Proyecto</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionGenerar">
   <property name="text">
    <string>Generar</string>
   </property>
  </action>
  <action name="acGen_Informe">
   <property name="text">
    <string>Generar Informe</string>
   </property>
  </action>
  <action name="acGen_Ultimo">
   <property name="text">
    <string>Generar √öltimo</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionAcerca_de_QT">
   <property name="text">
    <string>Acerca de QT</string>
   </property>
  </action>
  <action name="actionGuardar_Como">
   <property name="text">
    <string>Guardar Como</string>
   </property>
  </action>
  <action name="actionModo_Simple">
   <property name="text">
    <string>Modo Simple</string>
   </property>
  </action>
  <action name="actionModo_Completo">
   <property name="text">
    <string>Modo Completo</string>
   </property>
  </action>
  <action name="actionPersonalizar">
   <property name="text">
    <string>Personalizar</string>
   </property>
  </action>
  <action name="actionNuevo_Proyecto">
   <property name="text">
    <string>Nuevo Proyecto</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
